### ISSUE TITLE
Fix lookup_netgr_step()  not to access result of sysdb_getnetgr() in case of ret-val ENOENT

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#2385]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/2381
- Created at 2014-07-11 18:01:48 by [preichl](https://pagure.io/user/preichl)
- Closed at 2020-03-11 16:06:01 as Invalid
- Assigned to nobody
---

Fix lookup_netgr_step() respectively subsequently called check_cache not to access result of sysdb_getnetgr() in case of ENOENT is returned. 

Accessing output parameters of function which didn't returned EOK is dangerous practice.

*Not to block completely efforts on 1991 for the time being 'save' result is returned even if return value is ENOENT.*

### Comments

---

**Comment from [preichl](https://pagure.io/user/preichl) at [2014-07-17 15:32:32](https://pagure.io/SSSD/sssd/issue/2381#236591)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD 1.13 beta
rhbz:  => 0

---

**Comment from [preichl](https://pagure.io/user/preichl) at [2014-09-26 18:50:00](https://pagure.io/SSSD/sssd/issue/2381#236592)**

Fields changed

mark:  => 0

---

**Comment from [preichl](https://pagure.io/user/preichl) at [2015-02-10 14:37:20](https://pagure.io/SSSD/sssd/issue/2381#236593)**

Fields changed

milestone: SSSD 1.13 beta => SSSD 1.13 backlog
priority: major => minor

---

**Comment from [preichl](https://pagure.io/user/preichl) at [2015-02-12 20:27:35](https://pagure.io/SSSD/sssd/issue/2381#236594)**

Mass-moving tickets not planned for the next two releases.

Please reply with a comment if you disagree about the move..

milestone: SSSD 1.13 backlog => SSSD 1.15 beta

---

**Comment from [preichl](https://pagure.io/user/preichl) at [2017-02-24 14:54:10](https://pagure.io/SSSD/sssd/issue/2381#236595)**

**Metadata Update from @preichl**:
- Issue marked as blocked by: #1991
- Issue set to the milestone: SSSD Future releases (no date set yet)

---

**Comment from [preichl](https://pagure.io/user/preichl) at [2020-03-11 15:31:40](https://pagure.io/SSSD/sssd/issue/2381#632504)**

**Metadata Update from @thalman**:
- Custom field blocking reset (from 1991)
- Custom field design_review adjusted to on (was: 0)
- Custom field mark adjusted to on (was: 0)
- Custom field patch adjusted to on (was: 0)
- Custom field review adjusted to on (was: 0)
- Custom field sensitive adjusted to on
- Custom field testsupdated adjusted to on (was: 0)
- Issue close_status updated to: Invalid
- Issue status updated to: Closed (was: Open)

---

**Comment from [preichl](https://pagure.io/user/preichl) at [2020-03-11 15:32:01](https://pagure.io/SSSD/sssd/issue/2381#632505)**

No longer relevant due to switch to cache_req.

---

**Comment from [preichl](https://pagure.io/user/preichl) at [2020-03-11 15:32:02](https://pagure.io/SSSD/sssd/issue/2381#632506)**

**Metadata Update from @pbrezina**:
- Custom field blocking adjusted to 1991
- Issue marked as blocking: #1991
- Issue status updated to: Open (was: Closed)

---

**Comment from [preichl](https://pagure.io/user/preichl) at [2020-03-11 16:06:03](https://pagure.io/SSSD/sssd/issue/2381#632561)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: Invalid
- Issue status updated to: Closed (was: Open)
